lib/maru_swagger/config_struct.ex

```diff
@@ -13,13 +14,15 @@ defmodule MaruSwagger.ConfigStruct do
     force_json     = opts |> Keyword.get(:force_json, false)
     pretty         = opts |> Keyword.get(:pretty, false)
     swagger_inject = opts |> Keyword.get(:swagger_inject, []) |> Keyword.put_new_lazy(:basePath, base_path_func(module)) |> check_swagger_inject_keys
+    info           = opts |> Keyword.get(:info, []) |> check_info_inject_keys
 
     %__MODULE__{
       path: path,
       module: module,
       force_json: force_json,
       pretty: pretty,
       swagger_inject: swagger_inject,
+      info: info
```

Comment:
I think it's keep a `,` at the end of the line, so that the people who add another line shouldn't add `,` for this line 🙂